cmd/event-received/sirius_event_handler.go

```diff
@@ -104,24 +104,28 @@ func handleFeeApproved(ctx context.Context, client dynamodbClient, event events.
 		return err
 	}
 
+	if donor.Tasks.PayForLpa.IsCompleted() {
```

Comment:
```suggestion
	if donor.Tasks.PayForLpa.IsCompleted() || donor.Tasks.PayForLpa.IsApproved() {
```

since I've added that status